src/MultiMap.ts

```diff
@@ -1,7 +1,7 @@
 import '@dojo/shim/Symbol';
 import Map from '@dojo/shim/Map';
-import { from as arrayFrom } from '@dojo/shim/array';
-import { forOf, Iterable, IterableIterator, ShimIterator } from '@dojo/shim/iterator';
+import {from as arrayFrom} from '@dojo/shim/array';
```

Comment:
spaces between braces, spaces between braces, spaces between braces (see it rhymes)